h2o-bindings/bin/custom/python/gen_stackedensemble.py

```diff
@@ -45,9 +45,22 @@ def metalearner(self):
         >>> stack_blend.train(x=x, y=y, training_frame=train, blending_frame=blend)
         >>> stack_blend.metalearner
```

Comment:
can you please fix the doc at the same time? This was wrong from the beginning, this is not a property but a function:
```
stack_blend.metalearner()
```

Reply:
sure